client/js/array-util.js

```diff
@@ -1,7 +1,7 @@
 const getRange = function(fromNum, toNum){
 	return Array.from({length: toNum-fromNum+1}, (unused,i) => i+fromNum);
 };
-const getLetterRange = function(firstLetter, numLetters){
+const getLetterRange = function(firstLetter='A', numLetters=4){
```

Comment:
Set your text editor to use 2-space tabs instead of 4-space tabs; ask me if you have any trouble. This is key for readability, especially when you checked your code into a central repo that other devs will work off of.